lib/benchee/conversion/count.ex

```diff
@@ -151,7 +161,7 @@ defmodule Benchee.Conversion.Count do
       iex> Benchee.Conversion.Count.format(45.6789)
       "45.68"
 
-      iex> Benchee.Conversion.Count.format({45.6789, :thousand})
+      iex> Benchee.Conversion.Count.format({45.6789, %Benchee.Conversion.Unit{long: "Thousand", magnitude: "1_000", short: "K"}})
```

Comment:
Maybe

```
iex> thousand = %Benchee.Conversion.Unit{long: "Thousand", magnitude: "1_000", short: "K"}
iex> Benchee.Conversion.Count.format({45.6789, thousand})
```
or if `unit_for` were public
```
iex> thousand = Benchee.Conversion.Count.unit_for(:thousand)
iex> Benchee.Conversion.Count.format({45.6789, thousand})
```

I can't remember where this particular path is used off the top of my head, but it might be worth re-working the api so that the old doctest sill runs. Users should probably be able to deal with units in atoms only.